session.go

```diff
@@ -345,6 +353,8 @@ func ParseURL(url string) (*DialInfo, error) {
 	safe := Safe{}
 	for _, opt := range uinfo.options {
 		switch opt.key {
+		case "ssl":
+			ssl = opt.value == "true"
```

Comment:
It would be best to use [strconv.ParseBool()](https://golang.org/pkg/strconv/#ParseBool) to cover values like `ssl=TRUE` or `ssl=1`

Reply:
Well I'm not sure since the [specification](https://docs.mongodb.com/manual/reference/connection-string/#urioption.ssl) doesn't include  `TRUE` or `1`. 

Is this parameter also interpreted by the mongo server ? If yes and if we go for accepting `TRUE` and `1`, we have to be sure the server also accepts those values.

Reply:
If we do the same as you're doing now it will still send the server the correct value, but will make it less error prone for mgo users:

```go
if v, err := strconv.ParseBool(opt.value); err == nil && v {
	ssl = true
}
```

Reply:
done 450f038